### PR TITLE
[14.0][MIG] sale_stock_operating_unit: Migration to 14.0

### PR DESCRIPTION
**Pull request by dannyadair**
Supersedes https://github.com/OCA/operating-unit/pull/507

Depends on https://github.com/OCA/operating-unit/pull/464

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Hi @dannyadair I think the commit history is fine now :)

**Review by AaronHForgeFlow — COMMENTED** [review]
functional LG. 

I see know that the changes here: https://github.com/OCA/operating-unit/pull/377 are present so I will redo the history again.

**Comment by AaronHForgeFlow** [comment]
> functional LG.
> 
> I see know that the changes here: #377 are present so I will redo the history again.

I just made @RujiaL as co-author of the migration commit. I think it was not worth it to have 3 different commits for the migration. I also took the chance to update the licence Eficent -> ForgeFlow

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

**Review by JuanyDForgeflow — APPROVED** [review]

File: sale_stock_operating_unit/readme/CONFIGURATION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Go to 'Settings / Technical / Actions / User-defined Defaults' and remove
+the default set for the Shop.
```
Comment (francesco-ooops) [inline sale_stock_operating_unit/readme/CONFIGURATION.rst]: @AaronHForgeFlow is this still necessary?

File: sale_stock_operating_unit/static/description/icon.png
Comment (francesco-ooops) [inline sale_stock_operating_unit/static/description/icon.png:1]: is there an issue here?

File: sale_stock_operating_unit/readme/CONFIGURATION.rst (hunk 1 above)
Comment (AaronHForgeFlow) [reply sale_stock_operating_unit/readme/CONFIGURATION.rst]: Not sure, I've never use that feature. I have to check

File: sale_stock_operating_unit/static/description/icon.png
Comment (AaronHForgeFlow) [reply sale_stock_operating_unit/static/description/icon.png:1]: I think so, just added oca icon there

File: sale_stock_operating_unit/readme/CONFIGURATION.rst (hunk 1 above)
Comment (AaronHForgeFlow) [reply sale_stock_operating_unit/readme/CONFIGURATION.rst]: I checked and I think that is not necessary

**Comment by AaronHForgeFlow** [comment]
@francesco-ooops  ready to go?

**Review by francesco-ooops — APPROVED** [review]
LGTM

**Pull request merged**